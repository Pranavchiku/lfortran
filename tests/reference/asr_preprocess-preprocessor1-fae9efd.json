{
    "basename": "asr_preprocess-preprocessor1-fae9efd",
    "cmd": "lfortran --indent --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor1.f90",
    "infile_hash": "a07132d08deb1a0de21dd365371df337fe07652c9d42aa4c0a7e19ab",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor1-fae9efd.stdout",
    "stdout_hash": "4c74e23214e8520c8bda2d80d4732e64dafca92d99b1a98fc3cbeabf",
    "stderr": "asr_preprocess-preprocessor1-fae9efd.stderr",
    "stderr_hash": "99c0b0692bbaafa89731ca7e47e66b3253e32c81c341989ec4b74b01",
    "returncode": 0
}