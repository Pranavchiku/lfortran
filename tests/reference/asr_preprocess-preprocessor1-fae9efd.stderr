warning: Implied save feature
 --> tests/preprocessor1.f90:4:1
  |
4 | integer :: x, y
  | ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/preprocessor1.f90:4:1
  |
4 | integer :: x, y
  | ^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
